src/edwards/point.rs

```diff
@@ -29,6 +30,17 @@ impl EcPoint<Scalar> for Point {
     fn is_zero(&self) -> bool {
         self.c.x.is_zero() && !self.c.y.is_zero() && self.c.t.is_zero() && !self.c.z.is_zero()
     }
+    // based on https://tools.ietf.org/html/rfc8032#section-5.2.2
+    fn serialize(&self, _: bool) -> Vec<u8> {
+        let x = &self.c.x;
+        let y = &self.c.y;
+        // negative == odd
+        let x_0 = (((x.sgn0_le()>>1)&0x01)<<7) as u8;
+        let mut enc = y.to_bytes_be();
```

Comment:
In the encoding method for EdDSA, the number of bytes is `\ceil( (log(p) + 1)/8)`. Note that the +1 is part of the numerator.
For example. for P-256, a compressed point takes 33 bytes. but for Ed25519, it takes 32.

Reply:
So you mean that we need to check the length of enc?

Reply:
I think this is handled in `to_bytes_be()`, no?